### DISCUSSION TITLE
Release ?

### DISCUSSION THREAD

**Discussion by SebSept** [friends-of-presta/fop_console#237] [General]
Bon, il semble que pas grand monde ai du temps de dispo pour s'occuper de ce projet.

Je propose de release pour que les derniers changements soient accessibles à tous.

On fait ça ? 
Qui s'y colle ?

**Comment by ghost** [comment]
Je vais essayer de la faire d'ici demain ou avant fin de semaine

**Reply by ghost** [reply]
Je vais regarder les PR en cours avec deux validations  également

**Reply by ghost** [reply]
Et mettre à jour les labels :)
